2021-11-14 21:37:14,857 INFO    MainThread:4004 [wandb_setup.py:_flush():71] setting env: {}
2021-11-14 21:37:14,857 INFO    MainThread:4004 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-14 21:37:14,857 INFO    MainThread:4004 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ruben\VDoomLearning\src\wandb\run-20211114_213714-4cb1atpv\logs\debug.log
2021-11-14 21:37:14,857 INFO    MainThread:4004 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ruben\VDoomLearning\src\wandb\run-20211114_213714-4cb1atpv\logs\debug-internal.log
2021-11-14 21:37:14,858 INFO    MainThread:4004 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001827B355E80>
2021-11-14 21:37:14,859 INFO    MainThread:4004 [wandb_init.py:init():390] calling init triggers
2021-11-14 21:37:14,859 INFO    MainThread:4004 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-14 21:37:14,859 INFO    MainThread:4004 [wandb_init.py:init():435] starting backend
2021-11-14 21:37:14,860 INFO    MainThread:4004 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2021-11-14 21:37:14,869 INFO    MainThread:4004 [backend.py:ensure_launched():199] starting backend process...
2021-11-14 21:37:14,979 INFO    MainThread:4004 [backend.py:ensure_launched():205] started backend process with pid: 6052
2021-11-14 21:37:14,979 INFO    MainThread:4004 [wandb_init.py:init():444] backend started and connected
2021-11-14 21:37:14,992 INFO    MainThread:4004 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-14 21:37:14,993 INFO    MainThread:4004 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-14 21:37:14,993 INFO    MainThread:4004 [wandb_init.py:init():503] updated telemetry
2021-11-14 21:37:15,080 INFO    MainThread:4004 [wandb_init.py:init():533] communicating current version
2021-11-14 21:37:19,674 INFO    MainThread:4004 [wandb_init.py:init():538] got version response 
2021-11-14 21:37:19,674 INFO    MainThread:4004 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-14 21:37:20,293 INFO    MainThread:4004 [wandb_init.py:init():576] starting run threads in backend
2021-11-14 21:37:20,728 INFO    MainThread:4004 [wandb_run.py:_console_start():1693] atexit reg
2021-11-14 21:37:20,729 INFO    MainThread:4004 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-14 21:37:20,732 INFO    MainThread:4004 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-14 21:37:20,734 INFO    MainThread:4004 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-14 21:37:20,734 INFO    MainThread:4004 [wandb_init.py:init():603] run started, returning control to user process
2021-11-14 21:37:20,735 INFO    MainThread:4004 [wandb_run.py:_config_callback():911] config_cb None None {}
2021-11-14 21:40:46,070 INFO    MainThread:4004 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-14 21:40:46,071 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:43:36,030 INFO    MainThread:4004 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-14 21:43:36,040 INFO    MainThread:4004 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-14 21:43:36,040 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:43:53,171 INFO    MainThread:4004 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-14 21:43:53,884 INFO    MainThread:4004 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-14 21:43:53,884 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:44:03,756 INFO    MainThread:4004 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-14 21:44:04,524 INFO    MainThread:4004 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-14 21:44:04,525 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:46:33,062 INFO    MainThread:4004 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-14 21:46:33,069 INFO    MainThread:4004 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-14 21:46:33,069 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:46:38,251 INFO    MainThread:4004 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-14 21:46:41,384 INFO    MainThread:4004 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-14 21:46:41,384 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:46:47,426 INFO    MainThread:4004 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-14 21:46:48,031 INFO    MainThread:4004 [wandb_run.py:_config_callback():911] config_cb None None {}
2021-11-14 21:50:53,294 INFO    MainThread:4004 [wandb_run.py:finish():1347] finishing run teccr/VDoomLearning/4cb1atpv
2021-11-14 21:50:53,294 INFO    MainThread:4004 [jupyter.py:save_history():429] not saving jupyter history
2021-11-14 21:50:53,294 INFO    MainThread:4004 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-14 21:50:53,294 INFO    MainThread:4004 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-14 21:50:53,294 INFO    MainThread:4004 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-14 21:50:53,295 INFO    MainThread:4004 [wandb_run.py:_restore():1635] restore
2021-11-14 21:50:54,031 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 1102
}

2021-11-14 21:50:57,728 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 1102
}

2021-11-14 21:50:58,264 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7212
}

2021-11-14 21:50:58,376 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:58,484 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:58,594 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:58,701 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:58,810 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:58,920 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:59,029 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:59,139 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1102
  total_bytes: 7455
}

2021-11-14 21:50:59,249 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4364
  total_bytes: 7455
}

2021-11-14 21:50:59,357 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7455
  total_bytes: 7455
}

2021-11-14 21:50:59,466 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7455
  total_bytes: 7455
}

2021-11-14 21:50:59,575 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7455
  total_bytes: 7455
}

2021-11-14 21:50:59,682 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7455
  total_bytes: 7455
}

2021-11-14 21:50:59,972 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7455
  total_bytes: 7455
}

2021-11-14 21:51:00,625 INFO    MainThread:4004 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7455
  total_bytes: 7455
}
local_info {
}

2021-11-14 21:51:01,769 INFO    MainThread:4004 [wandb_run.py:_append_history():2011] rendering history
2021-11-14 21:51:01,769 INFO    MainThread:4004 [wandb_run.py:_append_summary():1966] rendering summary
2021-11-14 21:51:01,770 INFO    MainThread:4004 [wandb_run.py:_append_files():2061] logging synced files
